libs/s3/s3_blackbox_test_lib.py

```diff
@@ -211,16 +206,17 @@ def configre_minio_cloud(minio_repo=None,
             LOGGER.info("Installing minio client in current machine.")
             ACCESS = kwargs.get("access", None)
             SECRET = kwargs.get("secret", None)
-            run_local_cmd("wget {}".format(minio_repo))
+            run_local_cmd(f"wget {minio_repo}")
             run_local_cmd("chmod +x {}".format(os.path.basename(minio_repo)))
             run_local_cmd("./{} config host add s3 {} {} {} --api S3v4".format(os.path.basename(
                 minio_repo), endpoint_url, ACCESS, SECRET))
             for crt_path in minio_cert_path_list:
                 if not os.path.exists(crt_path):
-                    run_local_cmd("yes | cp -r {} {}".format(s3_cert_path, crt_path))
+                    run_local_cmd(f"yes | cp -r {s3_cert_path} {crt_path}")
             LOGGER.info("Installed minio client in current machine.")
 
             return True
+        # pylint: disable=broad-except
```

Comment:
Instead of disabling this warning, it would be a better idea to fix it by narrowing the scope of the Exceptions we are expecting in the except block

Reply:
Added expected error.